templates/documents.hbs

```diff
@@ -0,0 +1,7 @@
+<div class="documents component">
+  <ul>
```

Comment:
The title of the list is missing from Contentful. Please add the title as look in the design

Reply:
done